Algorithmia/datafile.py

```diff
@@ -115,6 +129,16 @@ def putFile(self, path):
                 raise raiseDataApiError(result)
             else:
                 return self
+    def putNumpy(self, array):
+        # Post numpy array as json payload
+        np_loader = pkgutil.find_loader('numpy')
+        if np_loader is not None:
+            import numpy as np
+            encoded_array = array.tolist()
+            self.putJson(encoded_array)
+            return self
+        else:
+            raise DataApiError("Attempted to .getNumpy() file without numpy available, please install numpy.")
```

Comment:
The exception message should say "Attempted to .putNumpy()" instead of ".getNumpy()"

Reply:
good catch, I'll correct that.